src/w3c/text-quote-selector.js

```diff
@@ -17,13 +17,13 @@ export default class TextQuoteSelector {
     let checkPrefix = this.prefix.replace(this.text, templateWord)
     let checkSuffix = this.suffix.replace(this.text, templateWord)
 
-    let fullText = `${checkPrefix}<span class="alpheios_worditem_incontext">${this.text}</span>${checkSuffix}`
+    let fullText = `${checkPrefix} <span class="alpheios_worditem_incontext">${this.text}</span> ${checkSuffix}`
```

Comment:
I think we're likely going to have to make some changes to this for RTL languages, but let's do that later.  I will enter an issue to make sure we cover it.

Reply:
ok - I think it is not really important for checking TQ to be equal, I am comparing it by fullText only because it needs less code than compare part by part (suffix, prefix, text).

But we have `contextHTML` method here - I think we will need to make it text direction dependent by language.